plugins/who-can.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: who-can
+spec:
+  version: "v0.1.0-alpha.1"
+  homepage: https://github.com/aquasecurity/kubectl-who-can
+  shortDescription: |
+    Shows which users, groups and service accounts can perform a given action
+  description: |
+    Shows who has permissions to VERB [TYPE | TYPE/NAME | NONRESOURCEURL] 
+    
+    VERB is a logical Kubernetes API verb like 'get', 'list', 'watch', 'delete', etc.
+    TYPE is a Kubernetes resource. Shortcuts, such as 'pod' or 'po' will be resolved. NAME is the name of a particular Kubernetes resource.
+    NONRESOURCEURL is a partial URL that starts with "/".
+  platforms:
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: https://github.com/aquasecurity/kubectl-who-can/releases/download/v0.1.0-alpha.1/kubectl-who-can_darwin_x86_64.tar.gz
+      sha256: "dcbe0048165e274b10ef0e8f77a9627d22a922465bf95b27bba7d1a37fc8963d"
+      files:
+        - from: kubectl-who-can
```

Comment:
[irrelevant to this PR]
cc: @corneliusweig this file is supporting the point for making .files[] optional (https://github.com/kubernetes-sigs/krew/issues/135), as we can copy out everything from these platform-specific tarballs, and then the plugin author needs to just specify the `bin` correctly.